documentation/release_notes.rst

```diff
@@ -8,6 +8,49 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.3.0
+===============
+
+New Features
+------------
+- Added dynamic selection as an experimental feature. Provides functions such as ``select``,
+  ``submit`` and ``submit_and_wait``. It includes selection policies: ``fixed_resource_policy``,
+  ``round_robin_policy``, ``dynamic_load_policy``, and ``auto_tune_policy``.
+  And provides default support for SYCL queues. 
```

Comment:
I'm confused by this wording, "And provides default support for SYCL queues." is a sentence fragment that likely needs to be joined with the previous sentence, removing the "and" before ``auto_tune_policy``.

Reply:
Thanks, Dan. Added your suggestion.

Reply:
I think that is correct but it would be good for @AnuyaWelling2801 @egfefey or @vossmjp to confirm this edit portrays the intended meaning.

Reply:
Looks ok to me with Dan's suggestions added.